service/src/main/java/uk/gov/hmcts/reform/fpl/service/payment/PaymentService.java

```diff
@@ -79,16 +79,16 @@ public void makePaymentForC2(Long caseId, CaseData caseData) {
 
         CreditAccountPaymentRequest paymentRequest = getCreditAccountPaymentRequest(caseId,
             c2DocumentBundle.getPbaNumber(),
-            c2DocumentBundle.getClientCode(),
-            defaultCustomerReferenceIfBlank(c2DocumentBundle.getFileReference()),
+            defaultParameterValueIfBlank(c2DocumentBundle.getClientCode()),
+            defaultParameterValueIfBlank(c2DocumentBundle.getFileReference()),
             localAuthorityName,
             feesData);
 
         callPaymentsApi(paymentRequest);
     }
 
-    private String defaultCustomerReferenceIfBlank(final String currentValue) {
-        return isNotBlank(currentValue) ? currentValue : BLANK_CUSTOMER_REFERENCE_VALUE;
+    private String defaultParameterValueIfBlank(final String currentValue) {
```

Comment:
Hmm, really don't like this name but can't think of a better alternative.

Reply:
We could go with the generic approach and copy `StringUtils` and just have `defaultIfBlank`

Reply:
infact why are we not using that already

Reply:
cheers guys.. changed it to StringUtils now.

Reply:
This can be removed now by the looks of it.

Reply:
This can be now removed no?